src/types.ts

```diff
@@ -40,6 +43,23 @@ export type AlertOptions = {
        */
       duration: number;
     }
+  | {
+      preset: "custom";
+      /**
+       * The name of an iOS-only SF Symbol. For a full list, see https://developer.apple.com/sf-symbols/.
+       * @platform ios
+       */
+      iosIconName?: string;
```

Comment:
Could we change this to have this format?

```ts
toast({ preset: 'custom', icon: { ios: { name, color } } })
```

This can get passed down as strings to Swift, but it would be better on the JS side to have an object like that I think.